### PR TITLE
Add `compile-proto-file` tool

### PR DESCRIPTION
**Pull request by intractable**
This PR adds a command line tool which allows compilation of `.proto` files from the command line. It is the same tool which has existed in the `grpc-haskell` repository for some time, but it belongs in this repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,17 +12,42 @@ This library provides a higher-level API to
...
-```
+```bash
 $ virtualenv pyenv
 $ source pyenv/bin/activate
 $ pip install protobuf==3.0.0b3  # Need the latest version for the newest protoc
 ```
 
 `brew install python` may also work.
+
+Alternate, the `nix-shell` environment provides a complete incremental build
```
Comment (ixmatus) [inline README.md]: `s/Alternate/Alternately/`

File: README.md [hunk 2]
```diff
@@ -12,17 +12,42 @@ This library provides a higher-level API to
...
+and test environment. From the root fo this repository:
```
Comment (ixmatus) [inline README.md]: `s/fo/of/`

File: README.md (hunk 1 above)
Comment (intractable) [inline README.md]: Thanks =)

File: README.md (hunk 2 above)
Comment (intractable) [inline README.md]: Thanks =)

**Pull request merged**